packages/calypso-e2e/src/lib/flows/start-site-flow.ts

```diff
@@ -154,6 +192,22 @@ export class StartSiteFlow {
 		await this.page.waitForSelector( selectors.themePickerContainer );
 	}
 
+	/**
+	 * Validates we've landed on the design setup screen used in the vertical flow.
+	 */
+	async validateOnDesignSetupScreen(): Promise< void > {
+		await this.page.waitForSelector( selectors.designSetupContainer );
```

Comment:
**suggestion:blocking**: While `waitForSelector` is perfectly adequate, for new code it would be preferable to use the `Locator` API instead. 

It could be a bit hard to initially wrap your head around, but [Locators](https://playwright.dev/docs/locators) are potentially more reliable as the target(s) are resolved at the time of calling.

Reply:
Thank you for the tip! I was wondering about the whole `waitForSelector` vs `locator` thing because I was seeing both used almost interchangeably. I'm learning so much :)